docs/admin/backup/restore.md

```diff
@@ -48,6 +48,52 @@ In the event of disaster, you can restore from a backup by doing the following:
 
 1. Wait until the `Restore` status is `Completed` and all `kcm` components are up and running.
 
+## If the Restore fails

```

Comment:
A note not to forget: the official way described here https://velero.io/docs/main/file-system-backup/#customize-restore-helper-container is no use since the erroneous code precedes this `ConfigMap` processing, hence this is no use, and the only solution here is to rename the `Deployment`

The problem is that `velero` (upstream) is incorrectly trying to identify the default image of the velero server image, fetching the hardcoded name of velero `Deployment` object. Hence, this section should be named similarly to `Known (temp) issue`, maybe give a reference to the bug https://github.com/vmware-tanzu/velero/issues/9023, because the reproducibility of the bug is 100%.